lib/docker-compose.base.yml

```diff
@@ -7,17 +7,19 @@ services:
         image: "${IMAGE}"
         container_name: sharelatex
         volumes:
-            - "${SHARELATEX_DATA_PATH}:/var/lib/sharelatex"
+            - "${OVERLEAF_DATA_PATH}:${OVERLEAF_IN_CONTAINER_DATA_PATH}"
         ports:
-            - "${SHARELATEX_LISTEN_IP:-127.0.0.1}:${SHARELATEX_PORT:-80}:80"
+            - "${OVERLEAF_LISTEN_IP:-127.0.0.1}:${OVERLEAF_PORT:-80}:80"
         environment:
           GIT_BRIDGE_ENABLED: "${GIT_BRIDGE_ENABLED}"
           GIT_BRIDGE_HOST: "git-bridge"
           GIT_BRIDGE_PORT: "8000"
           REDIS_HOST: "${REDIS_HOST}"
           REDIS_PORT: "${REDIS_PORT}"
-          SHARELATEX_MONGO_URL: "${MONGO_URL}"
-          SHARELATEX_REDIS_HOST: "${REDIS_HOST}"
+          OVERLEAF_MONGO_URL: "${MONGO_URL}"
+          OVERLEAF_REDIS_HOST: "${REDIS_HOST}"
```

Comment:
Do we need these two entries still with the lines below in place now?

Reply:
Correct, that's a leftover. Thanks!